examples/build.gradle

```diff
@@ -29,6 +29,7 @@ dependencies {
     implementation "io.grpc:grpc-protobuf:${grpcVersion}"
     implementation "io.grpc:grpc-stub:${grpcVersion}"
     compileOnly "javax.annotation:javax.annotation-api:1.2"
+    compile "org.apache.logging.log4j:log4j-api:2.11.2"
```

Comment:
We started using `implementation` as replacement for `compile` in examples.

Reply:
What's the difference?

Reply:
I've heard `compile` is deprecated.

Reply:
Change made